solidity/contracts/Oracle.sol

```diff
@@ -80,9 +75,7 @@ contract Oracle is Ownable {
 
   // MODIFIERS
 
-  modifier hasInternalId(
-    uint256 _internalId
-  ) {
+  modifier hasInternalId(uint256 _internalId) {
```

Comment:
I referenced the solidity styleguide for this: http://solidity.readthedocs.io/en/v0.4.23/style-guide.html

Seemed like overkill to have a one parameter method/modifier take 3 lines.